core-api-library/library/src/main/java/net/gini/android/core/api/requests/Exceptions.kt

```diff
@@ -0,0 +1,6 @@
+package net.gini.android.core.api.requests
+
+import java.io.IOException
+
+class ApiException(message: String) : IOException(message)
```

Comment:
Please return the response status code, headers and response body string in the `ApiException`.